### DISCUSSION TITLE
Change components' font from Venera to Torus or Torus (alternative) for the osu! argon and osu! argon pro skin

### DISCUSSION THREAD

**Discussion by Johnny555GH** [ppy/osu#22799] [Ideas]
The new osu! argon and argon pro skin have the Torus font, including the song progress component, but the scoreboard, accuracy counter, combo counter, and other components have the Venera font. I just don't think that matches, so let's change the components' font to Torus or the alternative version of Torus.

**Comment by mk56-spn** [comment]
you will probably be able to switch between the osu fonts on any component at some point, pretty sure there has been discussion about this somewhere already.

**Reply by bdach** [reply]
Also argon has a completely separate set of UI components anyway.

**Reply by mk56-spn** [reply]
Right, i meant for those i assume will just be converts. such as UR and pp counters
